src/Morphir/Visual/Components/TabsComponent.elm

```diff
@@ -16,14 +59,8 @@ type alias TabsComponentConfig msg =
     }
 
 
-type alias Tab msg =
-    { name : String
-    , content : Element msg
-    }
-
-
-tabsComponent : TabsComponentConfig msg -> Element msg
-tabsComponent config =
+view : TabsComponentConfig msg -> Element msg
```

Comment:
Can you add a few words as doc here. The Elm tooling requires docs for every exposed function (it doesn't complain right now because this is not part of the exposed modules).